sample_db_utils/core/driver.py

```diff
@@ -0,0 +1,205 @@
+#
+# This file is part of Sample Database Utils.
+# Copyright (C) 2019 INPE.
+#
+# Sample Database Utils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+
+"""This file contains Brazil Data Cube drivers to list the sample and store in database."""
+
+import logging
+import os
+from abc import abstractmethod, ABCMeta
+from copy import deepcopy
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from osgeo import ogr, osr
+import pandas as pd
+from geoalchemy2 import shape
+from geopandas import GeoDataFrame
+from shapely.geometry import Point
+from shapely.wkt import loads as geom_from_wkt
```

Comment:
There are some imports that is not being used like ``geom_from_wkt``, ``logging``, ``FileStorage``, etc.

Reply:
ok